app_dart/lib/src/request_handlers/get_benchmarks.dart

```diff
@@ -81,17 +160,17 @@ class GetBenchmarks extends RequestHandler<Body> {
         values.add(value);
       }
 
-      benchmarks.add(<String, dynamic>{
-        'Timeseries': <String, dynamic>{
-          'Timeseries': series,
-          'Key': keyHelper.encode(series.key)
-        },
-        'Values': values,
-      });
+      map['${series.taskName}.${series.label}'] = Result(series, values);
     }
-
-    return Body.forJson(<String, dynamic>{
-      'Benchmarks': benchmarks,
-    });
+    return map;
   }
 }
+
+class Result {
```

Comment:
Should we place this class in models?

Reply:
This class is used to hold temporary results, without a general-wide usage purpose across the whole app. We may just keep it here loally.

Reply:
In that case please document its purpose.

Reply:
Added.